bin/node/src/commands/net.rs

```diff
@@ -59,14 +60,15 @@ impl NetCommand {
 
         let rpc_config = Option::<RpcBuilder>::from(self.rpc);
 
+        let alloy_chain: alloy_chains::Chain = args.l2_chain_id.into();
```

Comment:
That's fine - i was hoping the l2 chain id would be updated to `Chain` itself but can be a follow on pr